victim-loss-identifier/src/agent.ts

```diff
@@ -87,24 +105,103 @@ export function provideHandleAlert(): HandleAlert {
               alertEvent.blockNumber!
             ))
           );
-          break;
-        
-        case "SCAM-DETECTOR-ICE-PHISHING":
-          findings.push();
-          break;
+        } else if (
+          // Tentative logic to keep track of scammers involved in multiple scam types
+          scammersCurrentlyMonitored[scammerAddress].firstAlertIdAppearance !== alertEvent.alertId! &&
+          !scammersInvolvedInMultipleScamTypes.includes(scammerAddress)
+        ) {
+          scammersInvolvedInMultipleScamTypes.push(scammerAddress);
+          console.log(`Scammer ${scammerAddress} has been involved in multiple scam types`);
+        }
+        break;
 
-        default:
-          return findings;
-      }
-    }
+      case "SCAM-DETECTOR-ICE-PHISHING":
+        const sourceAlertBotId = alertEvent.alert.source?.sourceAlert?.botId;
+        const sourceAlertHash = alertEvent.alert.source?.sourceAlert?.hash;
+        const underlyingAlert = sourceAlertHash
+          ? (
+              await getAlerts({
+                alertHash: sourceAlertHash,
+                blockNumberRange: {
+                  // Requires blockNumberRange to be set to fetch the alert successfully
+                  startBlockNumber: DEFAULT_FORTA_API_QUERY_BLOCK_NUMBERS.START,
+                  endBlockNumber: alertEvent.blockNumber
+                    ? alertEvent.blockNumber
+                    : DEFAULT_FORTA_API_QUERY_BLOCK_NUMBERS.END,
+                },
+              })
+            ).alerts[0]
```

Comment:
Yes, because we're basically expecting one result, as we're querying per alert hash. Even if more than one alerts are returned, those will be identical (just like the alert from the Chainpatrol bot that was duplicated)